src/plugins/chatwizard/chatwizard.js

```diff
@@ -262,11 +298,20 @@ var componentName = "wb-chtwzrd",
 			}
 		} );
 
+		// On chat reset button pressed: toggle experience to wizard
+		$document.on( "click", selector + "-container .btn-reset", function( event ) {
```

Comment:
You can do this now because you have declared $reset as part of my previous comment.
```suggestion
		$reset.on( "click", function( event ) {
```